server/lib/util.py

```diff
@@ -41,11 +45,14 @@
     'poverty': ['USA', 'India'],
 }
 
+_root_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
should we put this in a more general util and use it?

Reply:
Good idea. Made a function, but still can not share with nl_server for now.